sapi/fpm/fpm/fpm_conf.c

```diff
@@ -915,8 +915,7 @@ static int fpm_conf_process_all_pools(void)
 			}
 
 			if (config->listen_backlog < FPM_BACKLOG_DEFAULT) {
-				zlog(ZLOG_WARNING, "[pool %s] listen.backlog(%d) was too low for the ondemand process manager. I updated it for you to %d.", wp->config->name, config->listen_backlog, FPM_BACKLOG_DEFAULT);
-				config->listen_backlog = FPM_BACKLOG_DEFAULT;
+				zlog(ZLOG_WARNING, "[pool %s] listen.backlog(%d) is too low for the ondemand process manager. I suggest updating it to at least %d.", wp->config->name, config->listen_backlog, FPM_BACKLOG_DEFAULT);
```

Comment:
we should probably verify why this is an issue for ondemand mode and test it with lower listen backlog. This was added together with ondemand mode from checking the changes so there might be a reason for adding it. There should be also a test added for setting it low and using ondemand. I would prefer to separate it to a new PR as it's unrelated to other change.